### DISCUSSION TITLE
maxlength character counter colors `red` to display only when characters reach 100% limit

### DISCUSSION THREAD

**Discussion by miteyema** [laravel/nova-issues#5068] [Ideas & Feature Requests]
- Laravel Version: 9.40.1
- Nova Version: 4.19.0
- PHP Version: 8.1.11
- Database Driver & Version: mariadb  Ver 15.1 Distrib 10.8.3-MariaDB
- Operating System and Version: macOS 12.6
- Browser type and version: Firefox 106.0.5 (64-bit)
- Reproduction Repository: n/a

### Description:
```php
use Laravel\Nova\Fields\Text;

Text::make(__('QUESTION'), 'question')
                    ->rules('required', 'max:65')
                    ->maxlength(65)
                    ->enforceMaxlength()
```

Please also see [maxlength documentation](https://nova.laravel.com/docs/4.0/resources/fields.html#setting-maxlength-on-text-fields)

### Detailed steps to reproduce the issue on a fresh Nova installation:

- Start typing text in the input field.
- When typing 46 characters, the '46/65' character counter turn **yellow**, that's fine.
- When typing 59 characters, the '59/65' character counter turns **red**, but the user is still allowed to type more characters (up to 65 in this case).

It seems confusing to show a red character counter to the user when the user is still allowed to type more characters.
It would be preferable not to use any red and just stay with the yellow color, as nothing is actually wrong.

**Comment by crynobone** [comment]
Unable to reproduce:

https://user-images.githubusercontent.com/172966/202067412-1dbcdfba-1224-4c9d-948a-ed9f28f6a089.mp4

**Comment by miteyema** [comment]
@crynobone Thanks for the quick reply. 

Could you retry with a much lager `maxlength` value as per my description?
Please find below a video where the character counter turns red before the actual 65 character limit.

https://user-images.githubusercontent.com/2599590/202076697-0e4a68f7-6f5e-4907-81fe-d724c04e921f.mov

**Comment by crynobone** [comment]
![image](https://user-images.githubusercontent.com/172966/202077747-979b6903-3148-48fe-a59d-fc9ee25b057b.png)

There no different

**Comment by crynobone** [comment]
The warning indicator is based "yellow" for above 60% limit and "red" as above 90% limit. 

Marking this as feature request.
